### ISSUE TITLE
Error When Clicking Activate Instantiate Object on Animator Character Model Script

### ISSUE THREAD

**Issue by SpikedBatGames** [suriyun-production/mmorpg-kit-docs#609]
InspectorButton: Unable to find method ActivateInstantiateObject in MultiplayerARPG.AnimatorCharacterModel
UnityEngine.Debug:LogWarning(Object)
InspectorButtonPropertyDrawer:OnGUI(Rect, SerializedProperty, GUIContent) (at Assets/UnityMultiplayerARPG/Core/UnityEditorUtils/Editor/InspectorButtonPropertyDrawer.cs:27)
UnityEditor.EditorGUILayout:PropertyField(SerializedProperty, Boolean, GUILayoutOption[])
BaseCustomEditor:SetPropertyFieldVisibilities(SerializedProperty) (at Assets/UnityMultiplayerARPG/Core/UnityEditorUtils/Editor/BaseCustomEditor.cs:146)
BaseCustomEditor:OnInspectorGUI() (at Assets/UnityMultiplayerARPG/Core/UnityEditorUtils/Editor/BaseCustomEditor.cs:109)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr)

**Comment by insthync** [comment]
Oh, thank you for bug report, changes codes in `BaseCharacterModel.cs` line `22-27` to 
```
        [InspectorButton(nameof(SetEquipmentContainersBySetters))]
        public bool setEquipmentContainersBySetters;
        [InspectorButton(nameof(DeactivateInstantiatedObjects))]
        public bool deactivateInstantiatedObjects;
        [InspectorButton(nameof(ActivateInstantiatedObject))]
        public bool activateInstantiateObject;
```

**Comment by insthync** [comment]
Fixed in 1.59
